stream-chat-android-client/src/main/java/io/getstream/chat/android/client/models/Message.kt

```diff
@@ -93,4 +94,4 @@ public data class Message(
     @SerializedName("reply_to_message")
     var replyTo: Message? = null
 
-) : CustomObject
+) : CustomObject, Serializable
```

Comment:
It's not right. We can't expose serializable mechanics min, such model, only for UI reason. Also, it could be even dangerous. Message class contains non-serializable fields (e.g. `List<Attachment>, List<User>` at least). It could be dangerous (I think we can get crash if the app is restored by Android on such a screen). How did it work before???
I think the proper way is to store message-id and extract it from the repo